content/en/apps/concepts/interoperability.md

```diff
@@ -4,13 +4,35 @@ linkTitle: "Interoperability"
 weight: 8
 description: >
   Exchanging information between CHT Core and other systems
-keywords: interoperability integrations
+keywords: interoperability integrations fhir icd openhie openhim
 relatedContent: >
   apps/features/integrations
 ---
 
-*Interoperability* and *Integration* are often used interchangeably but mean different things. [This article](https://www.linkedin.com/pulse/lets-talk-interoperability-vs-integration-mike-fleck) offers a useful delineation between the two.
+# Introduction
 
-With **interoperability**, systems work together, even if they weren’t designed specifically to work together, because standard communication protocol techniques are applied. Two (or more) systems that weren't designed to be interoperable require **integration**.
+Interoperability is the ability of health information systems to work together, even if they weren't specifically designed to work together. With interoperability, patient information can be seen, exchanged, and used across different platforms. This is different from _integration_ which requires custom development to connect two specific systems together. 
+
+Interoperability is the best practice for health systems because it allows information from one system to be shared with one or more other systems for with no additional development.
+
+# Components
+
+Useful components and reference information for interoperability include
+
+- [OpenHIE](https://ohie.org/) defines the architecture for an interoperability layer.
+- [OpenHIM](http://openhim.org/) is a middleware component designed to ease interoperability between systems.
+- [HL7 FHIR](https://www.hl7.org/fhir/index.html) is a messaging format to allow all systems to understand the format of the message.
+- [ICD-11](https://www.who.int/standards/classifications/classification-of-diseases) and [LOINC](https://loinc.org/) are examples of classification systems.
+
+# CHT
+
+The structure of documents in the CHT database reflect the configuration of the system, and therefore do not map directly to a FHIR message format. To achieve interoperability you should use middleware to convert the CHT datastructure into a standardized form so the other systems can read it.
+
+{{< figure src="../interoperability.png" link="../interoperability.png" >}}
```

Comment:
not sure if it's documented, but existing docs image protocol is to create a directory (eg `interoperability`) and then put the image in there. You'll have to update the `src` to match.  See here [for example](https://github.com/medic/cht-docs/blame/interoperability-best-practice/content/en/apps/concepts/workflows.md#L22).

Reply:
Done.